fcos-policy-engine/src/main.rs

```diff
@@ -108,8 +110,8 @@ pub(crate) async fn pe_serve_graph(
     let wariness = compute_wariness(&query);
     ROLLOUT_WARINESS.observe(wariness);
 
-    // TODO (zonggen): remove hard-coded empty graph and use the graph fetched from fcos-graph-builder
-    let cached_graph = graph::Graph::default();
+    let query_str = serde_qs::to_string(&query).unwrap_or_default();
+    let cached_graph = utils::fetch_graph_from_gb(query_str).await?;
 
     let arch_graph = policy::pick_basearch(cached_graph, basearch)?;
```

Comment:
This looks like redundant as the graph builder already filtered the basearch.

Reply:
I agree, it should be quick to check that dropping it should not change the resulting graph.

Reply:
Diff shows no content change :)